docs/sources/operators-guide/configure/configuring-tsdb-block-upload.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Configuring TSDB block upload"
+menuTitle: "Configuring TSDB block upload"
+description: "Learn how to configure Grafana Mimir to enable TSDB block upload"
+weight: 120
+---
+
+# Configuring TSDB block upload
+
+Grafana Mimir supports uploading of historic TSDB blocks, sourced from f.ex. Prometheus, Thanos, or
+Grafana Mimir itself.
+
+**Note**: Please read about limitations related to importing blocks from Thanos in
+[Migrating from Thanos or Prometheus to Grafana Mimir]({{< relref "../../migration-guide/migrating-from-thanos-or-prometheus.md" >}}).
+
+The functionality is disabled by default, but can be enabled via the `-compactor.block-upload-enabled`
+CLI flag, or via the corresponding `limits.compactor_block_upload_enabled` configuration parameter:
+
+```yaml
+limits:
+  # Enable TSDB block upload
+  compactor_block_upload_enabled: true
+```
+
+## Enabling TSDB block upload per tenant
+
+If your Grafana Mimir has multitenancy enabled, you can still use the preceding method to enable
+TSDB block upload for all tenants. If instead you wish to enable it per tenant, you can use the
+runtime configuration to set a per-tenant override:
+
+1. Enable [runtime configuration]({{< relref "about-runtime-configuration.md" >}}).
+1. Add an override for the tenant that should have TSDB block upload enabled:
+
+```yaml
+overrides:
+  tenant1:
+    compactor_block_upload_enabled: true
+```
```

Comment:
Can we add new section to this document listing current known limitations of the blocks upload feature?

- results-cache needs flushing
- uploaded blocks with time range within `-querier.query-store-after` period are not queryable (because we only query ingesters)
- inability to upload Thanos blocks

Reply:
Done, PTAL.